tests/subsys/rtio/rtio_api/src/rtio_iodev_test.h

```diff
@@ -58,6 +58,23 @@ static void rtio_iodev_timer_fn(struct k_timer *tm)
 	struct rtio_iodev_sqe *iodev_sqe = data->iodev_sqe;
 	struct rtio_iodev *iodev = (struct rtio_iodev *)iodev_sqe->sqe->iodev;
 
+	if (data->sqe->op == RTIO_OP_RX) {
+		uint8_t *buf;
+		uint32_t buf_len;
+
+		int rc = rtio_sqe_rx_buf(iodev_sqe, 16, 16, &buf, &buf_len);
+
+		if (rc != 0) {
+			printk("Error getting buffer\n");
```

Comment:
You can use TC_PRINT here in place of printk, as this could be considered part of the test suite and ztest is included.

I know this iodev_test should probably be made into more of a mock object of sorts one day.

Reply:
I meant to delete it and missed it :)